Automattic/sensei
# Translate emails

Resolves #6627.

## Proposed Changes
Translates strings in all emails.

## Testing Instructions


1. Enable emails feature.
2. Run `npm run i18n:build` to update the .pot file.
3. Use a plugin like [Loco Translate](https://en-ca.wordpress.org/plugins/loco-translate/) to translate some of the email text to another language.
4. Change your Site Language to the language you provided translations for.
5. Trigger each email and ensure that the content is translated. In the screenshot below I translated "Course Name" to Spanish:

![Screenshot 2023-03-23 at 7 45 59 AM](https://user-images.githubusercontent.com/1190420/227194133-c15270cf-bf37-414e-8fa1-59e361d86872.jpg)

## Pre-Merge Checklist
